--- a/vendor/alom/graphviz/src/Alom/Graphviz/BaseInstruction.php	2025-04-18 18:31:30.116061980 +0200
+++ b/vendor/alom/graphviz/src/Alom/Graphviz/BaseInstruction.php	2025-04-18 23:34:07.836164265 +0200
@@ -46,11 +46,16 @@
      */
     protected function escape($value)
     {
+dpplog(5, "in escape: value: $value");
+dpplog(5, "in escape: print_r(value): " . print_r($value,true));
+        $x = ($this->needsEscaping($value)) ? '"' . str_replace('"', '""', str_replace('\\', '\\\\', $value)) . '"' : $value;
+dpplog(5, "in escape: value2: $x");
         return ($this->needsEscaping($value)) ? '"' . str_replace('"', '""', str_replace('\\', '\\\\', $value)) . '"' : $value;
     }
 
     protected function escapePath(array $path)
     {
+dpplog(9, "in escapePath: value: zzzzz");
         $list = array();
         foreach ($path as $element) {
             $list[] = $this->escape($element);
